docs/index.md

```diff
@@ -18,6 +18,7 @@ These are the main benefits of Thinking Cities platform:
 Thinking Cities provides the following APIs:
 
 - [Authentication API](authentication_api.md): manages tokens for APIs usage.
+- [Authorization with grants API](authorization_api.md): manages grants.
```

Comment:
Maybe it should be added also to mkdocs.yml so the file also appears in the RTD rendering of the doc at http://thinking-cities.readthedocs.io/en/master/

Reply:
done in https://github.com/telefonicaid/thinking-cities/pull/130/commits/433f12ba35f6b2c426d4735f99a98c1e45a4fd9e